canary/umbrel-app.yml

```
@@ -0,0 +1,29 @@
+manifestVersion: 1
+id: canary
+category: bitcoin
+name: Canary
+version: "1.0.0"
+tagline: Early warning system for Bitcoin cold storage
+description: >-
+  A canary in the cold mine. When your bitcoins are in cold storage, you seldom check on them.
+  Canary acts as an early warning system that alerts you the moment your coins move, giving you
+  immediate notification of any activity on your wallets. Built with Rust and BDK, it provides
+  real-time transaction intelligence with advanced pattern recognition (RBF fee bumping, CPFP
+  acceleration) and instant multilingual notifications via ntfy.sh. Set custom balance alerts
+  (equal, less than, or more than) to monitor your holdings. Perfect for monitoring cold storage
+  and Bitcoin holdings you don't actively use.
```

Re-format description for a clearer structure.
```suggestion
description: >-
  A canary in the cold mine. When your bitcoins are in cold storage, you seldom check on them.


  Canary acts as an early warning system that alerts you the moment your coins move, giving you immediate notification of any activity on your wallets.


  Built with Rust and BDK, it provides real-time transaction intelligence with advanced pattern recognition (RBF fee bumping, CPFP acceleration) and instant multilingual notifications via ntfy.sh.


  Set custom balance alerts (equal, less than, or more than) to monitor your holdings. Perfect for monitoring cold storage and Bitcoin holdings you don't actively use.
```